### PR TITLE
fix: incomingpath on codespaces

### PR DESCRIPTION
**Pull request by danyhoron**
upgrade tasks.json to version 2.0.0

### CODE REVIEW

File: src/namespaceDetector.ts [hunk 1]
```diff
@@ -33,9 +33,9 @@ export default class NamespaceDetector {
 
         const csprojFile = csprojs[0];
         const fileContent = await this.read(Uri.file(csprojFile));
-        const rootNamespace = new CsprojReader(fileContent).getRootNamespace();
+        let rootNamespace = new CsprojReader(fileContent).getRootNamespace();
         if (rootNamespace === undefined) {
-            return undefined;
+            rootNamespace = path.basename(csprojFile, '.csproj')
```
Comment (PoLaKoSz) [inline src/namespaceDetector.ts:38]: This edit is familiar but why is it in this PR?

Comment (danyhoron) [reply src/namespaceDetector.ts]: you're right.
While debugging, I noticed that I got some weird namespaces and that fixed it.
I reverted.

Comment (PoLaKoSz) [reply src/namespaceDetector.ts]: Ah, okey. For some reason the `let rootNamespace` didn't reverted back to `const rootNamespace`. Can You fix it, please?

**Comment by KreativJos** [comment]
I have no way of testing this in codespaces currently, so I cannot test for that.
All default functions still do work, so it is fine by me.

**Pull request merged**